### ISSUE TITLE
docs(svelte): Svelte subscription docs out-of-date for >=2.0.0

### ISSUE THREAD

**Issue by qbunt** [urql-graphql/urql#2992]
From [this section of the docs](https://formidable.com/open-source/urql/docs/advanced/subscriptions/#svelte)

```
<script>
  import { gql, getContextClient, subscriptionStore } from '@urql/svelte';

  const messages = subscriptionStore({
    client: getContextClient(),
    query: gql`
      subscription MessageSub {
        newMessages {
          id
          from
          text
        }
      }`
    });
</script>

{#if !$messages.data}
  <p>No new messages</p>
{:else}
  <ul>
    {#each $messages.data as message}
      <li>{message.from}: "{message.text}"</li>
    {/each}
  </ul>
{/if}

```

> As we can see, the $messages.data is being updated and transformed **by the handleSubscription function**. This works over time, so as new messages come in, we will append them to the list of previous messages.

---

There is no `handleSubscription` function in the previous code example, and it's not obvious, in Svelte, how you could implement one as you would need to subscribe to the subscriptionStore writable store itself (just kind of confusing and unnecessary).

Digging around a little bit, `subscriptionStore` takes an optional second handler parameter, which works to demonstrate having a handler function.

There are potentially 2 solutions for correcting the docs: 
- Mention the second subscriptionHandler in _this_ section of the docs, (so you don't have to go clicking through another section) and show how you can use it (what I came to the docs to find)
- Remove the mention and rely on the click-through to the next section of the docs

Given that the other code examples have something kind of similar, I think it makes sense to mention, it's otherwise pretty buried. 

Thanks!

**Comment by kitten** [comment]
Yes, the new Svelte docs are partly still lagging behind the new implementation. Obviously also tagging your discussion thread here (https://github.com/urql-graphql/urql/discussions/2993) since it's related.

This is basically leftover work from: https://github.com/urql-graphql/urql/pull/2370

**Comment by qbunt** [comment]
@kitten would you want a PR to update some of the docs? I don't want to step on toes

**Comment by kitten** [comment]
@qbunt I'd love a PR for that! 🙌 We really just didn't get to scanning all docs for the old API, so I don't think anyone else is working on this right now

**Comment by qbunt** [comment]
should be able to put something together later. The new API is way simpler, but handling subscriptions is something we do a lot. I'll keep you posted

**Comment by kitten** [comment]
Just a quick update; the `handleSubscription` function used to exist, and we don't actually touch the code of our bindings all that often. What happened was that after the last set of refactors after suggestions from our Svelte users and community, the refactor wasn't quite right.

To be exact, we had accidentally implemented the `handleSubscription` function on the `mutationStore` instead of the `subscriptionStore`, which is obviously completely non-sensical.

I've only discovered this now because we're adding TSDocs to _all_ of our APIs going forward and to our existing packages and public API.

It was a trivial fix and has been merged: https://github.com/urql-graphql/urql/pull/3078
However, as this was pretty unexpected, it's now subject to our next merge window, which is a major release merge window (well, luckily, after all, this is a blunder, but a breaking change at that)
I don't think we'll have to revert the docs, since it seems the Svelte GraphQL Subscriptions doc wasn't correct either, since it still referred to `handleSubscription` in some parts, but other parts were updated to remove it.

Anyway, this will be readded in the next release, and the TSDocs will document it sufficiently.
